plugins/inputs/webhooks/plex/README.md

```diff
@@ -0,0 +1,45 @@
+# plex webhooks
+
+You should configure Plex's Webhooks to point at the `webhooks` service. To do this, follow the instructions in the [Plex webhooks support documentation](https://support.plex.tv/articles/115002267687-webhooks/). Webhooks are a premium feature and require an active Plex Pass Subscription for the Plex Media Server account.
+
+## Events
+
+The titles of the following sections are links to the full payloads and details for each event. The body contains what information from the event is persisted. The format is as follows:
+```
+# TAGS
+* 'tagKey' = `tagValue` type
+# FIELDS
+* 'fieldKey' = `fieldValue` type
+```
+The tag values and field values show the place on the incoming JSON object where the data is sourced from.
+
+**Tags:**
+* 'event' = `event.event` string
+* 'is_user_webhook' = `event.user` bool
+* 'is_owner_webhook' = `event.owner` bool
+* 'user_id' = `event.Account.id` string
+* 'user_name' = `event.Account.title` string
+* 'server_title' = `event.Server.title` string
+* 'server_uuid' = `event.Server.uuid` string
```

Comment:
Consider omitting `server_uuid` and `player_uuid`, unless they are typically displayed items.